ntp-daemon/src/main.rs

```diff
@@ -18,6 +18,9 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
     let mut config = Config::from_args(args.config, args.peers).await?;
 
+    // Warn/error if the config is unreasonable.
+    config.check();
```

Comment:
Kind of thinking this check should be split/also/only happen after tracing is fully initialized, if your config has a valid sentry config, I'm assuming that the current no peers setup warning wouldn't show up in there right now, only in local logs (as long as you configured the command line log level high enough).

Reply:
Fair point, fixed.